JSONAPI/Http/QueryableToManyRelatedResourceDocumentMaterializer.cs

```diff
@@ -46,13 +46,5 @@ public abstract class QueryableToManyRelatedResourceDocumentMaterializer<TRelate
         /// </summary>
         protected abstract Task<IQueryable<TRelated>> GetRelatedQuery(string primaryResourceId, CancellationToken cancellationToken);
 
-        /// <summary>
-        /// If the client doesn't request any sort expressions, these expressions will be used for sorting instead.
-        /// </summary>
-        /// <returns></returns>
-        protected virtual string[] GetDefaultSortExpressions()
```

Comment:
The more I think about it... it's may needed and overriden by custom subclasses.
I've seen the same method in MappedDocumentMaterializer.
Is this it needed?